api/base/serializers.py

```diff
@@ -653,7 +676,8 @@ def to_representation(self, obj, envelope='data'):
                         data['embeds'][field.field_name] = result
                 else:
                     try:
-                        data['relationships'][field.field_name] = field.to_representation(attribute)
+                        if not (is_anonymous and hasattr(field, 'view_name') and field.view_name.find('users:') != -1):
```

Comment:
Whhaaaa? Why don't you like… no, that sounds like a good plan. I lean towards doing a mixin instead. Does that suit you?